src/bpftrace.cpp

```diff
@@ -1282,6 +1286,24 @@ uint64_t BPFtrace::resolve_kname(const char *name)
   return addr;
 }
 
+uint64_t BPFtrace::resolve_cgroupid(const char *path)
+{
+  uint64_t cgid;
+  int err, mount_id;
+  file_handle *handle = reinterpret_cast<struct file_handle *>(
+    malloc(sizeof(file_handle) + sizeof(uint64_t)));
```

Comment:
Can handle just be allocated on the stack instead here?

Reply:
Oh, I see it needs more space than the struct now...
Could we do something like create our own wrapper struct with the regular file_handle and a uint64_t inside to avoid the dynamic allocation then? Or use smart pointers - I just don't like having malloc and free in here

Reply:
ok, I'll rewrite it to put it on the stack.